docs/Usage-Guide.md

```diff
@@ -1,3 +1,177 @@
+
+   - Optionally, clone the git repository and build from the source files.
+   - ` git clone -v "https://github.com/Kentico/xperience-by-kentico-sitecore-migration-tool.git" "{YOUR PATH}"`
+
+2. Modify the console application's configuration file `/examples/Migration.Toolkit.Sitefinity.Console/appsettings.json`
+   - See [Configuration File Options](#configuration-file-options)
+3. Modify code for any additional customizations
+   - Entity Framework is included. Add additional custom models to Migration.Toolkit.Sitefinity.Data.Core.EF.SitefinityContext for any custom tables. An additional import service will need to be created. See other import services under `/src/Migration.Toolkit.Sitefinity/Services`.
+   - Use the RestSdkBase when creating a provider that calls the rest api endpoints. This will ensure the rest client is running and provides methods to call the rest client.
+   - Add additional field types for custom field types. See `/src/Migration.Toolkit.Sitefinity/FieldTypes` for out-of-the-box field types in Sitefinity. Using the `IFieldType`, will automatically add it to the import processing. You can also use the `FieldTypeBase` to handle the defaults.
+4. Set up Sitefinity
+   - Ensure _Deployment Mode_ is set to _Source_. This feature will export the content type definitions from the database and store them on the disk. Like XbyK's Continuous Integration (CI) files. ![alt text](image-5.png)  
+     [More information in the Sitefinity docs.](https://www.progress.com/documentation/sitefinity-cms/deployment-mode)
+   - Once this is set, there is a menu item under _Administration_ for Export / Import ![alt text](image.png)
+   - Use Sitefinity's _Export for Deployment_ feature to create content type files. In the main navigation under _Administration » Export / Import_ Select _Export for Deployment_ from the left navigation and select _Export_. The path displayed here will be the value of `ModuleDeploymentFolderPath` in the `appsettings.json` file. Ensure you use the full path to the folder in the setting. The default value for this path is `~\App_Data\Sitefinity\Deployment`  
+     ![alt text](image-6.png)
+   - Check the default language for source Sitefinity site, it must match what is in XbyK. Look in _Administration >> Settings >> Basic >> Language_!  
+     ![alt text](image-2.png)
+   - Make sure the Sitefinity content API is set to allow all content and is enabled. View the web service configuration under _Administration >> Web services >> Default_  
+     ![alt text](image-4.png)  
+     In the configuration options for 'Default' make sure all content types are selected and 'This service is active' is checked/true.  
+      ![alt text](image-1.png)
+5. Set up XbyK
+   - Modify the XbyK default language to match the default language of the Sitefinity source site being migrated e.g. 'en-us'. Sitefinity's default English culture is 'en'.  

```

Comment:
What exactly is meant by the default language? I think this should refer to the primary language of a website channel: https://docs.kentico.com/developers-and-admins/configuration/website-channel-management#create-website-channels

Reply:
The default language is describing the English language in the Languages application in XbyK. The English language is there by default on a new install. The primary language of the channel is handled on import.

Reply:
The English language is there by default, but any Language can be set as the default language. This should be changed to "Modify the default 'English' language in XbyK..." if you mean the default English language.

Reply:
Ok this has been fixed in latest commit!